run.py

```diff
@@ -793,10 +793,11 @@ def run(self, outdir):
 
             if not check_value["failure"] and not check_value["skipped"]:
                 if not self.quiet:
-                    if os.path.basename(os.path.dirname(self.directory)) != "tests":
-                        path_name = os.path.join(os.path.basename(os.path.dirname(self.directory)), self.name)
+                    test_name_offset = self.directory.find("tests/")
```

Comment:
We cannot use `tests` in subdirectories then 🤔

Reply:
> We cannot use `tests` in subdirectories then 🤔

Why not?

Moving `tests/dns/*` into `tests/dns/tests/...` seems to work fine:
```
[jason@desktop …/dev/suricata/master]  master ❯ ~/oisf/dev/suricata-verify/master/run.py tests/dns
    Finished release [optimized] target(s) in 0.01s
Number of concurrent jobs: 8
===> dns/tests/dns-eve-empty-format: OK
===> dns/tests/dns-frames: OK
===> dns/tests/dns-query-name: OK
===> dns/tests/dns-invalid-opcode: OK
===> dns/tests/dns-answer-name: OK
```